core/dbt/include/global_project/macros/materializations/models/materialized_view/materialized_view.sql

```diff
@@ -38,14 +44,20 @@
     -- `BEGIN` happens here:
     {{ run_hooks(pre_hooks, inside_transaction=True) }}
 
-    -- cleanup
-    -- move the existing view out of the way
+    -- determine the scenario we're in: create, full_refresh, alter, refresh data
     {% if existing_relation is none %}
         {% set build_sql = get_create_materialized_view_as_sql(target_relation, sql) %}
-    {% elif full_refresh_mode or existing_relation.type != 'view' %}
+    {% elif full_refresh_mode or not existing_relation.is_view %}
         {% set build_sql = get_replace_materialized_view_as_sql(target_relation, sql, existing_relation, backup_relation, intermediate_relation) %}
+    {% elif config_updates and on_configuration_change == 'apply' %}
+        {% set build_sql = get_alter_materialized_view_as_sql(target_relation, config_updates, sql, existing_relation, backup_relation, intermediate_relation) %}
+    {% elif config_updates and on_configuration_change == 'skip' %}
+        {% set build_sql = 'select 1 as skip_configuration_changes where 0 = 1;' %}
```

Comment:
Is there a better way to pass a "no-op" sql string into the expected call below?